File: src/main/java/io/github/hdzitao/editstarters/initializr/Initializr.java
package io.github.hdzitao.editstarters.initializr;

/**
 * Initializr
 *
 * @version 3.2.0
 */
public interface Initializr {
    void initialize(InitializrRequest request, InitializrResponse response, InitializrChain chain) throws Exception;
}

File: src/main/kotlin/io/github/hdzitao/editstarters/initializr/Initializr.kt
package io.github.hdzitao.editstarters.initializr

/**
 * Initializr
 *
 * @version 3.2.0
 */
interface Initializr {
    fun initialize(request: InitializrRequest, response: InitializrResponse, chain: InitializrChain)
}
